### PR TITLE
Bump module to go1.14 & use vendored certificate conformance tests

### PR DESCRIPTION
**Pull request by dprotaso**
Thus the go commands will default to -mod=vendor

See: https://golang.org/doc/go1.14#go-command

### CODE REVIEW

**Comment by dprotaso** [comment]
```
can't load package: package knative.dev/serving/test/conformance/certificate/http01: cannot find package "." in:
	/home/prow/go/src/knative.dev/net-http01/vendor/knative.dev/serving/test/conformance/certificate/http01
```

**Comment by dprotaso** [comment]
I think you can't run tests from another module

**Comment by dprotaso** [comment]
so `pull-knative-net-http01-integration-tests` is not functional

File: test/conformance/certificate.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+limitations under the License.
+*/
+
+package conformance
+
+import (
+	"testing"
+
+	"knative.dev/serving/test/conformance/certificate/http01"
+	"knative.dev/serving/test/conformance/certificate/nonhttp01"
```
Comment (mattmoor) [inline test/conformance/certificate.go:22]: Format Go code:
```suggestion

```

**Comment by dprotaso** [comment]
@ZhiminXiang should be good now

**Comment by ZhiminXiang** [comment]
/lgtm
/approve

thanks for fixing it!

**Pull request merged**